### PR TITLE
Prepare for v0.15.0 release

### PR DESCRIPTION
**Pull request by sigma-andex**
This PR prepares elmish-enzyme for the Purescript v0.15.0 release.

### CODE REVIEW

**Comment by fsoikin** [comment]
Hey, thanks for the pull request! Do you mean to say that ES modules are finally coming? I'm so excited! Is there a specific timeline?

**Comment by sigma-andex** [comment]
Yes, they are coming and hopefully pretty soon. There will be one last v0.14.6 release and then there will be v0.15.0 with es modules.

File: test/Main.purs [hunk 1]
```diff
@@ -20,9 +20,9 @@ import Test.Spec.Runner (runSpec)
 foreign import _configureJsDomViaFfi :: Type
 
 main :: Effect Unit
-main = do
+main = launchAff_ $ do
   Enzyme.configure Adapter.unofficialReact_17
```
Comment (fsoikin) [inline test/Main.purs]: Now that `unofficialReact_17 :: Aff Adapter`, I don't think this line would compile.

Comment (sigma-andex) [reply test/Main.purs]: i was so focussed on getting it running in elmish that I forgot to update this test 😅. fixed now

**Review by fsoikin — COMMENTED** [review]
Ok, this looks good so far.

**Comment by sigma-andex** [comment]
Superseded by #16

**Pull request closed without merging**